nested-data-and-nested-loops/practice-nested-data-and-nested-loops.md

```diff
@@ -0,0 +1,516 @@
+# Practice: Nested Data and Nested Loops
+
+## Directions
+
+Complete all questions below.
+
+## Practice
+
+<!-- Question 1 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: short-answer
+* id: 8mavqk
+* title: Nested Data and Nested Loops
+##### !question
+
+What is the syntax to get `'6501 Island St. SE'` from `restaurants`?
+
+```python
+restaurants = [
+    {
+        "name": "Dough Zone",
+        "cuisine": "Chinese",
+        "address": "1299 Dough Ave N"
+    },
+    {
+        "name": "Bingos",
+        "cuisine": "Carribean",
+        "address": "6501 Island St SE"
+    },
+    {
+        "name": "Toyoda",
+        "cuisine": "Japanese",
+        "address": "12543 Sushi Way NE"
+    }
+]
+```
+
+Use `[]` syntax with double-quotes where needed.
+
+##### !end-question
+##### !answer
+
+restaurants[1]["address"]
+
+##### !end-answer
+##### !hint
+
+Use `[]` syntax with double-quotes where needed.
+
+Examples:
+
+- `some_list[0]`
+- `some_dictionary["some key"]`
+
+##### !end-hint
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- Question 2 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: code-snippet
+* language: python3.6
+* id: WfX3l1
+* title: Nested Data and Nested Loops
+### !question
+
+Implement `get_restaurant_addresses()`. This method takes in one parameter, `restaurants`. The method should return a list of strings, which are restaurant addresses.
```

Comment:
This sounds more natural to me to describe the desired contents of the result, followed by the type information clarification.
```suggestion
Implement `get_restaurant_addresses()`. This method takes in one parameter, `restaurants`. The method should return a list of restaurant addresses, which are strings.
```

If the other ordering is desired, then I'd insert 'the' (the current phrasing makes it sound like 'restaurant address' is a type).
```suggestion
Implement `get_restaurant_addresses()`. This method takes in one parameter, `restaurants`. The method should return a list of strings, which are the restaurant addresses.
```